Vwitter/View Controllers/AudioSegmentPickerViewController.m

```diff
@@ -0,0 +1,117 @@
+//
+//  AudioSegmentPickerViewController.m
+//  Vwitter
+//
+//  Created by Christina Li on 8/5/22.
+//
+
+#import "AudioSegmentPickerViewController.h"
+#import "SpotifyAPIManager.h"
+#import "VWHelpers.h"
+
+#import <SpotifyiOS/SpotifyiOS.h>
+#import <MARKRangeSlider/MARKRangeSlider.h>
+
+static CGFloat const kViewControllerRangeSliderWidth = 290.0;
+static CGFloat const kViewControllerLabelWidth = 100.0;
+
+@interface AudioSegmentPickerViewController ()
+
+@property (nonatomic, strong) MARKRangeSlider *rangeSlider;
+@property (nonatomic, strong) UILabel *label;
+@property (nonatomic) NSUInteger duration;
+
+@end
+
+@implementation AudioSegmentPickerViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[SpotifyAPIManager shared] playTrack:self.selectedTrack.uriString];
+    // Do any additional setup after loading the view.
+    self.title = @"Choose Song Section";
+    [self getSpotifyInfo];
+    
+}
+
+- (void)viewWillAppear:(BOOL)animated {
+    
+}
+
+- (void)getSpotifyInfo {
+    [[SpotifyAPIManager shared] getDurationWithCompletion:^(NSUInteger duration, NSError * _Nullable error) {
+        if (!error) {
+            NSLog (@"duration %lu", (unsigned long)duration);
+            self.duration = duration;
```

Comment:
`duration` is only ever used in `setUpViewComponents` right? So it would be simpler if you just make it an input parameter for that method, there's no need to make it an `@property`